tests/dummy/app/templates/application.hbs

```diff
@@ -1,9 +1,35 @@
-{{#materialize-navbar name='ember-cli-materialize'}}
-  <li>{{#link-to 'buttons'}}Buttons{{/link-to}}</li>
-  <li>{{#link-to 'navbar'}}Navbar{{/link-to}}</li>
-  <li>{{#link-to 'cards'}}Cards{{/link-to}}</li>
-  <li>{{#link-to 'collapsible'}}Collapsible{{/link-to}}</li>
-  <li>{{#link-to 'input'}}Input{{/link-to}}</li>
-{{/materialize-navbar}}
+<header>
+  {{#materialize-navbar name='ember-cli-materialize'}}
+    <li>{{#link-to 'buttons'}}Buttons{{/link-to}}</li>
+    <li>{{#link-to 'navbar'}}Navbar{{/link-to}}</li>
+    <li>{{#link-to 'cards'}}Cards{{/link-to}}</li>
+    <li>{{#link-to 'collapsible'}}Collapsible{{/link-to}}</li>
+    <li>{{#link-to 'input'}}Input{{/link-to}}</li>
+  {{/materialize-navbar}}
+</header>
+
+<main>
+  {{outlet}}
+</main>
+
+{{#materialize-footer}}
+  <div class="container">
+    <div class="row">
+      <div class="col l6 s12">
+        <h5 class="white-text">Footer Content</h5>
+        <p class="grey-text text-lighten-4">You can use rows and columns here to organize your footer content.</p>
+      </div>
+      <div class="col l4 offset-l2 s12">
+        <h5 class="white-text">Links</h5>
+        <ul>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 1</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 2</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 3</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 4</a></li>
+        </ul>
+      </div>
+    </div>
+  </div>
```

Comment:
@franktcurran I think that we should challenge every component in this library by asking the following question:

> How much easier/better does adding a given Ember component make it for developers, relative to the raw materializecss HTML?

In this case (and frankly in my proposed syntax as well - #29) we're saving developers a few lines of markup at most. Compare this to the value added to the library by the `{{materialize-input}}` component, which may be used hundreds of times.